runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -65,13 +77,27 @@ AntiDragConfig getConfig(ConfigManager configManager)
 	@Override
 	protected void startUp() throws Exception
 	{
+		if (client.getGameState() == GameState.LOGGED_IN)
+		{
+			clientThread.invokeLater(() ->
+			{
+				if (!config.onShiftOnly() && !inPvp())
+				{
+					client.setInventoryDragDelay(config.dragDelay());
+					setBankDragDelay(config.dragDelay());
+				}
+			});
+		}
+
 		keyManager.registerKeyListener(this);
 	}
 
 	@Override
 	protected void shutDown() throws Exception
 	{
 		client.setInventoryDragDelay(DEFAULT_DELAY);
+		setBankDragDelay(DEFAULT_DELAY);
```

Comment:
This actually looks like a bug fix? I may take this independently/it should be in a separate commit.

Reply:
Referring to the shutdown behavior? This really is there because the original anti-drag plugin had it. I will test to validate.